src/data.toml

```
@@ -1,13 +1,13 @@
 ["1196075698301968455"] # pyoncord, responses for triggered intents
 undetailed_question = "The information you provided is not detailed enough to answer your question.\nPlease provide more information or rephrase your question."
-get_addons = "Bunny is specifically designed for use with [Vendetta](<https://discord.gg/rJgnCJ3Kzm>) plugins:\n\n- You can retrieve plugin links from: [Bunny Plugin Links](https://bn-plugins.github.io/vd-web/). Requests to add additional plugin URLs to this list are welcome.\n\n- While Bunny supports Vendetta themes, you may also install themes made for [Enmity](<https://discord.gg/rMdzhWUaGT>) or [Revenge](<https://discord.com/invite/ddcQf3s2Uq>). Please note that compatibility with these themes is not guaranteed and older themes may not work properly.\n\nFor a curated list of broken plugins, see: [Broken Plugins List](https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596)"
+get_addons = "Bunny is combatible with Vendetta plugins:\n\n- You can retrieve plugin links from: [Bunny Plugin Links](https://bn-plugins.github.io/vd-web/). Requests to add additional plugin URLs to this list are welcome.\n\n- While Bunny supports [Vendetta](<https://discord.gg/rJgnCJ3Kzm>) themes, you may also install themes made for [Enmity](<https://discord.gg/rMdzhWUaGT>) or [Revenge](<https://discord.com/invite/ddcQf3s2Uq>). Please note that compatibility with these themes is not guaranteed and older themes may not work properly.\n\nFor a curated list of broken plugins, see: [Broken Plugins List](https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596)"
```

combatible is not a word, overall a lot of repetition 
```suggestion
get_addons = "Bunny is compatible with Vendetta plugins\n\n- You can retrieve those from https://bn-plugins.github.io/vd-web/.\n-# Requests to add additional plugin URLs to this list are welcome.\n\n- While Bunny supports [Vendetta](<https://discord.gg/rJgnCJ3Kzm>) themes, you may also install ones made for [Enmity](<https://discord.gg/rMdzhWUaGT>) or [Revenge](<https://discord.com/invite/ddcQf3s2Uq>). Please note that compatibility with these is not guaranteed and older themes may not work properly.\n\nFor a curated list of broken plugins, see https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
```